### DISCUSSION TITLE
Flow deck 2.0 not getting connected

### DISCUSSION THREAD

**Discussion by MAHASHANA** [bitcraze/discussions#1465] [Q&A]
Hello, I am having trouble connecting my Flow Deck. It's not being detected on either of my two Crazyflies, even though the AI Deck works fine on both drones.
SYS: Crazyflie 2.1 is up and running!
SYS: Build 0:e527895e8c6e (2023.11) CLEAN
SYS: I am 0x20303554513150070040005A and I have 1024KB of flash!
CFGBLK: v1, verification [OK]
DECK_CORE: 1 deck(s) found
DECK_CORE: Calling INIT on driver bcFlow2 for deck 0
ZR2: Z-down sensor [FAIL]
PMW: Motion chip id: 0x:0xFF
IMU: BMI088: Using I2C interface.
IMU: BMI088 Gyro connection [OK].
IMU: BMI088 Accel connection [OK]
IMU: BMP388 I2C connection [OK]
ESTIMATOR: Using Kalman (2) estimator
CONTROLLER: Using PID (1) controller
MTR-DRV: Using brushed motor driver
SYS: About to run tests in system.c.
SYS: NRF51 version: 2024.2 (CF21)
EEPROM: I2C connection [FAIL].
SYS: configblock [FAIL]
STORAGE: Storage check [OK].
IMU: BMI088 gyro self-test [OK]
Error while initializing the PMW3901 sensor
DECK_CORE: Deck 0 test [FAIL].
SYS: deck [FAIL]

**Comment by evoggy** [comment]
Hi! Sounds like it might be a hardware issue. Did it previously work and stopped or has it never worked?

**Reply by MAHASHANA** [reply]
Hello, it never worked from the beginning.  It worked fine with the first installation. Following that, it gets detected on rare occasions and frequently displays the error shown above.

**Reply by evoggy** [reply]
Ok, please drop us an email at support@bitcraze.io and we'll arrange a replacement.

**Reply by MAHASHANA** [reply]
That helps thank you for the assistance.
